shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/model/AuthTokenResponse.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.mark.kmmic.data.service.auth.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal data class AuthTokenResponse(
```

Comment:
I prefer to change to `AuthTokenApiModel` (e.g., https://developer.android.com/topic/architecture/data-layer#business-models)

Reply:
@suho updated at [b5c86d3](https://github.com/markgravity/kmm-ic/pull/67/commits/b5c86d34cf43b9e1cac1bdf1dc2ae0f36ddf159a)